common/lib/xmodule/xmodule/course_module.py

```diff
@@ -830,6 +830,13 @@ class CourseFields(object):
         },
         scope=Scope.settings
     )
+    allow_unsupported_xblocks = Boolean(
+        display_name=_("Allow the creation of unsupported XBlocks"),
+        help=_(
+            "If set to True, XBlocks that are not supported by edX can be created in Studio. Scary legal text..."
```

Comment:
I suggest ( @sstack22 please weigh in too)

"Enter true or false. If true, additional, you can add unsupported content types to your course in Studio. Scary legal text."

(In the past, I have been told in no uncertain terms that we cannot include "edX" even for things that truly are edX-specific, such as the text on the video upload page, but up to you!)